app/src/main/java/net/skyscanner/backpack/demo/compose/BpkStarRatingStory.kt

```diff
@@ -0,0 +1,175 @@
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.demo.R
+
+@Composable
+fun BpkStarRatingStory(
+  modifier: Modifier = Modifier,
+) {
+  val context = LocalContext.current
+  Box(
+    modifier = modifier.fillMaxHeight(),
+    contentAlignment = Alignment.Center,
+  ) {
+    Column(modifier = Modifier.padding(start = BpkSpacing.Lg)) {
+      BpkText(
+        text = stringResource(R.string.static_default_rating_title),
+        style = BpkTheme.typography.heading2,
+      )
+      Row() {
+        Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+          StaticRatingSample(
+            context = context,
+            size = BpkStarRatingSize.Large,
+            text = stringResource(R.string.icons_large),
+          )
+        }
+        Column() {
+          StaticRatingSample(
+            context = context,
+            size = BpkStarRatingSize.Small,
+            text = stringResource(R.string.icons_small),
+          )
+        }
+      }
+      Spacer(modifier = Modifier.padding(top = BpkSpacing.Lg))
+      Column() {
+        BpkText(
+          text = stringResource(R.string.hotel_star_rating_title),
+          style = BpkTheme.typography.heading2,
+        )
+        Row() {
+          Column(modifier = Modifier.padding(end = BpkSpacing.Xl)) {
+            HotelRatingSample(
+              context = context,
+              size = BpkStarRatingSize.Large,
+              text = stringResource(R.string.icons_large),
+            )
+          }
+          Column() {
+            HotelRatingSample(
+              context = context,
+              size = BpkStarRatingSize.Small,
+              text = stringResource(R.string.icons_small),
+            )
+          }
+        }
+      }
+    }
+  }
+}
+
+@Composable
+private fun HotelRatingSample(context: Context, size: BpkStarRatingSize, text: String) {
+  BpkText(
+    text = text,
+    style = BpkTheme.typography.heading3,
+  )
+  BpkHotelRating(
+    rating = 1f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 1f, 5),
+    size = size,
+  )
+  BpkHotelRating(
+    rating = 2f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 2f, 5),
+    size = size,
+  )
+  BpkHotelRating(
+    rating = 3f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 3f, 5),
+    size = size,
+  )
+  BpkHotelRating(
+    rating = 4f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 4f, 5),
+    size = size,
+  )
+  BpkHotelRating(
+    rating = 5f,
+    contentDescription = context.getString(R.string.star_rating_accessibility_status, 5f, 5),
+    size = size,
+  )
+}
+
+@Composable
+private fun StaticRatingSample(context: Context, size: BpkStarRatingSize, text: String) {
+  BpkText(
+    text = text,
+    style = BpkTheme.typography.heading3,
+  )
+  BpkStarRating(
```

Comment:
similarly here you could have a float list of the ratings you want to display and loop through them to avoid repetition